src/bitcoin-cli.cpp

```diff
@@ -538,7 +538,7 @@ class NetinfoRequestHandler : public BaseRequestHandler
         // Report detailed peer connections list sorted by direction and minimum ping time.
         if (DetailsRequested() && !m_peers.empty()) {
             std::sort(m_peers.begin(), m_peers.end());
-            result += strprintf("<->   type   net tp  mping   ping send recv  txn  blk  hb %*s%*s%*s ",
+            result += strprintf("<->   type   net  v  mping   ping send recv  txn  blk  hb %*s%*s%*s ",
```

Comment:
I think that if we don't write the "v", we can make the width of the added column 1 instead of 2.

Reply:
I like that extra space as is, IMHO it visually looks better.

![image](https://github.com/bitcoin/bitcoin/assets/4500994/6d01c577-c612-4189-8a0d-4f32eba8b785)

Reply:
ok, if it's on purpose it's fine.